src/absa/inference/api.py

```
@@ -0,0 +1,17 @@
+"""Public multi-aspect inference contract for ReviewPulse v3."""
+
+
+def normalise_aspects(aspects: list[str]) -> list[str]:
+    seen: set[str] = set(); result: list[str] = []
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)